lib/scim_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module ScimRails
-  VERSION = '0.4.0'
+  VERSION = '0.5.0'
```

Comment:
do you think this minor version update is reasonable?

Reply:
Yes! It doesn't introduced any breaking changes so we're good.